holmscan/controller.py

```diff
@@ -22,24 +23,18 @@
 logging.getLogger("anyconfig").setLevel(logging.ERROR)
 
 
-CONFIGURABLES = [
-    "HOLMSCAN_DEBUG",
-    "HOLMSEC_PASSWORD",
-    "HOLMSEC_URL",
-    "HOLMSEC_USERNAME",
-]
+CONFIGURABLES = ["HOLMSCAN_DEBUG", "HOLMSEC_ENDPOINT", "HOLMSEC_TOKEN"]
 DEFAULTS = {
     "HOLMSCAN_DEBUG": False,
-    "HOLMSEC_PASSWORD": "",
-    "HOLMSEC_URL": "https://sc.holmsecurity.com/",
-    "HOLMSEC_USERNAME": "",
+    "HOLMSEC_ENDPOINT": "https://se-api.holmsecurity.com/v1",
```

Comment:
The variable name `DEFAULTS` is to non descriptive. It do not tell what kind or defaults for what purpose.

Reply:
Leaving as is for now.